From: Jeremy Soller <jeremy@system76.com>
Date: Wed, 20 Jun 2018 19:22:06 +0200
Subject: Add support for GDM3 theme alternatives

 GNOME vanilla and systemd76 derivative ships their own GDM theme.
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=787454
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/gnome-shell/+bug/1715722
Last-Update: 2017-09-08
---
 js/ui/sessionMode.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/sessionMode.js b/js/ui/sessionMode.js
index 97118dc..9ac12d6 100644
--- a/js/ui/sessionMode.js
+++ b/js/ui/sessionMode.js
@@ -42,7 +42,7 @@ const _modes = {
 
     'gdm': {
         hasNotifications: true,
-        stylesheetName: 'ubuntu.css',
+        stylesheetName: 'gdm3.css',
         isGreeter: true,
         isPrimary: true,
         unlockDialog: imports.gdm.loginDialog.LoginDialog,
